optimum/utils/normalized_config.py

```diff
@@ -44,6 +44,9 @@ def with_args(cls, allow_new: bool = False, **kwargs) -> Callable[[PretrainedCon
         return functools.partial(cls, allow_new=allow_new, **kwargs)
 
     def __getattr__(self, attr_name):
+        if attr_name == "config":
+            return super().__getattr__(attr_name)
+
```

Comment:
Why?

Reply:
I had an infinte recursion error, which makes sense no? We call `self.config` in the `__getattr__` redefinition.